tests/test_mesh.py

```diff
@@ -343,23 +330,51 @@ def arithmetic_statmesh_setup(self):
 
     def test_mutliply_vector_by_scalar(self):
         self.arithmetic_mesh_vector_setup()
-        scalar = 12.0
-        test_vector_tag = self.mesh_1_vector.mesh.tag_get_handle(self.vector_tag_name)
-        test_data = self.mesh_1_vector.mesh.tag_get_data(test_vector_tag, self.volumes1)
+        scalar = 12.0 
         exp_res = [[12.0, 24.0], [18.0, 30.0], [-12.0, -18.0], [-24.0, -30.0]]
-        obs_res = scalar*test_data
+        obs_res = scalar*self.mesh_1_vector.testing[:] 
         assert_array_almost_equal(exp_res, obs_res)
     
     def test_multiply_vector_tag_by_scalar_tag(self):
+        self.arithmetic_mesh_vector_setup()
+        exp_res = [[12.0, 24.0], [9.0, 15.0], [-3.0, -4.5], [-3.0, -3.75]] 
+        scalar_data = self.mesh_1_vector.test_scalar_tag[:].reshape((4,1))
```

Comment:
This looks like a transpose. Maybe we can use that operator to avoid knowing the size?